docs/kb/semgrep-cloud-platform/missing-pr-commments.md

```diff
@@ -0,0 +1,50 @@
+---
+description: Use this reference to check why you may not be receiving Semgrep comments on PRs/MRs.
+tags:
+  - Semgrep CI
+  - PR Comments
+  - Rule Board
+---
+
+# Why am I not receiving PR/MR comments?
+
+If you have configured Semgrep in CI and Semgrep Cloud Platform to create comments when a rule generates a finding in a PR or MR, but you are not seeing those comments, check the following possibilities.
+
+## Are comments supported on your SCM?
+
+PR/MR Comments are currently supported for GitHub, GitLab, and Bitbucket. If you are using another SCM, comments are not yet supported. Bitbucket Data Center is also not supported yet.
```

Comment:
```suggestion
PR or MR comments are currently supported for:

* All GitHub plans
* All GitLab plans
* Bitbucket Cloud

PR or MR comments are not supported for:

* Bitbucket Data Center
* Azure Repos
* Any other SCM or repository provider
```

Suggesting to make it a list to make it easy to skim.

I suspect some variant of this information may live in the CI docs, so after you merge this I may turn this into a react component so we can import it (more a note for myself)

Reply:
yeah this is a good change, thank you!